misc/git/hooks/golangci-lint

```diff
@@ -17,13 +17,12 @@
 # We will enable it for everything here, but with most of the linters disabled.
 # See: https://github.com/vitessio/vitess/issues/5503
 
-# NOTE(sougou): tool is disabled till it's fixed
-exit
-
 GOLANGCI_LINT=$(command -v golangci-lint >/dev/null 2>&1)
 if [ $? -eq 1 ]; then
   echo "Downloading golangci-lint..."
   curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.21.0
 fi
 
-golangci-lint run --disable=ineffassign,unused,gosimple,staticcheck,errcheck,structcheck,varcheck,deadcode
+#golangci-lint run --disable=ineffassign,unused,gosimple,staticcheck,errcheck,structcheck,varcheck,deadcode
+
+golangci-lint run go/...
```

Comment:
does this block committing on any linter error? some of the other hooks (like staticcheck) give you the option of `ack` to skip fixing linter errors.

Reply:
It just gives out the error